### DISCUSSION TITLE
Add contribution graph to mobile

### DISCUSSION THREAD

**Discussion by vilewired** [community/community#57288] [Mobile]
Everything looks very clean, I am just missing my nice little green squares :(

**Comment by mxie** [comment]
Thanks for your feedback! What OS are you on? Android does not currently support contribution graphs, but iOS does have them as widgets for the logged-in user.

**Reply by vilewired** [reply]
Yeah I am on Android, I thought it would be a nice feature to add on android too :)

**Comment by ManishParashar0** [comment]
let check
